setup.py

```diff
@@ -331,8 +343,7 @@ def _populate_hdf5_info(dirstosearch, inc_dirs, libs, lib_dirs):
             sys.stdout.write("""
     HDF5_DIR environment variable not set, checking some standard locations ..\n""")
             for direc in dirstosearch:
-                sys.stdout.write('checking %s ...\n' % direc)
-                hdf5_version = check_hdf5version(os.path.join(direc, 'include'))
+                hdf5_version = get_hdf5version(os.path.join(direc, 'include'))
```

Comment:
This should have an underscore
```suggestion
                hdf5_version = get_hdf5_version(os.path.join(direc, 'include'))
```